src/appleseed.studio/main/main.cpp

```diff
@@ -280,15 +281,15 @@ int main(int argc, char* argv[])
     g_previous_message_handler = qInstallMsgHandler(message_handler);
 
     QApplication application(argc, argv);
+    // QApplication sets C locale to user's locale, we need to fix this
+    std::setlocale(LC_ALL, "C");
+
     QApplication::setOrganizationName("appleseedhq");
     QApplication::setOrganizationDomain("appleseedhq.net");
     QApplication::setApplicationName("appleseed.studio");
     QApplication::setApplicationVersion(Appleseed::get_lib_version());
     QApplication::setWindowIcon(QIcon(make_app_path("icons/appleseed.png")));
-
-    // The locale must be set after the construction of QApplication.
-    QLocale::setDefault(QLocale::C);
```

Comment:
Oops, you removed this, but we do need it: the Qt locale must be also set. Please keep this and just add your call after it.